src/css/styles.scss

```diff
@@ -1,46 +1,90 @@
-// Load CSS Reset from NPM
-// CSS resets "normalize" how browser styles are applied.
-// Try commenting this out and saving. What changes in the browser?
-@import "~normalize.css";
+@media screen and (min-width: 468px) and (max-width: 768px) {
+	.logo {
+		background-color: black;
+		font-size: 30px;
+		color: white;
+		padding: 20px;
+		margin: auto 0;
+		padding-bottom: 200px;
+		font-family: freight-big-pro, serif;
+		font-weight: bold;
+		font-style: normal;
+	}
+
+	.search {
+		position: absolute;
+		right: 60px;
+		top: 40px;
+	}
+
+	.menu {
+		position: absolute;
+		right: 20px;
+		top: 40px;
+	}
+
+	.mainpic {
+		display: inline-block;
+		vertical-align: middle;
+		text-align: center;
+		border-radius: 50px;
+	}
 
-// An example of a partial.
-// Check out the _typography.scss file to edit the CSS inside!
-@import "typography";
-@import "layout";
+	.drinkname {
+		font-family: freight-big-pro, serif;
```

Comment:
Is this font linked in your HTML or CSS anywhere? If not, I will probably not be able to see it.